chain/client-primitives/src/types.rs

```diff
@@ -216,7 +216,7 @@ pub fn format_shard_sync_phase(
     match &shard_sync_download.status {
         ShardSyncStatus::StateDownloadHeader => format!(
             "{} requests sent {}, last target {:?}",
-            paint("HEADER", Purple.bold(), use_colour),
+            paint("HEADER", Magenta.style().bold(), use_colour),
```

Comment:
lol, did they actually rename the color when bumping version or do you just like megenta more? :)

Reply:
ah nevemind, it's different library